examples/decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
path: ```examples/02_decoding/plot_haxby_decoder.py``` can be changed I guess.

For all examples: numbers are added to control the ordering.